src/serial/parse_input.py

```diff
@@ -1,6 +1,35 @@
-def parse_string(message: str) -> str:
+import json
+
+ACC_GYR_Value = {'x': -1, 'y': -1, 'z': -1}
+LID_Value = {'left': -1, 'middle': -1, 'right': -1}
+COM_BAT_Value = {'x': -1}
+RPM_TMP_WET_GPS_Value = {'x': -1, 'y': -1}
+
+Arrays = [['x', 'y', 'z'], ['left', 'middle', 'right']]
 
-    split_message = message.split("-")
-    type = split_message.pop(0)
+def dataUpdater(value, myJson, splitMessage, array, size):
```

Comment:
instead of passing size, can we just get it from splitMessage within this function